ui/lib/core/addon/components/ttl-form.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * @module TtlForm
+ * TtlForm components are used to enter a Time To Live (TTL) input.
+ * This component does not include a label and is designed to take
+ * a time and unit, and pass an object including seconds and
+ * timestring when those two values are changed.
+ *
+ * @example
+ * ```js
+ * <TtlForm @onChange={action handleChange} @unit={{m}}/>
```

Comment:
curious if the double curly brackets around m should be one?

Reply:
Should actually just be quotes, good catch